DataService.js

```diff
@@ -13,7 +13,8 @@ angular.module('PathOfDamage')
               total: 0
             },
             more: {
-              placeholder: "More Damage"
+              placeholder: "More Damage",
+              values: []
```

Comment:
None of the other tables in sections have values though, you just seemingly picked two at random to add it to? They are initialized online 163 fyi: `table.values = table.values || [];`